app/services/code_hosting/github_open_pr.rb

```diff
@@ -1,11 +1,29 @@
 module FastlaneCI
   # Encapsulates the data we need to know how to checkout a branch from a fork
   # We can probably use this for all the things, though.
+
+  # Example data
```

Comment:
Beautiful documentation in this class 👍

Reply:
Yesss, the way we'd want every class ideally 👍